### DISCUSSION TITLE
Need help with integration of few lines into the repo

### DISCUSSION THREAD

**Discussion by 6969111** [vercel/community#1084] [Help]
Hi.
I am not a coder.
I need help with these instructions:
https://vercel.com/docs/concepts/analytics/audiences/quickstart
I don't know how to add 2-3 code lines to my repo.
Can someone help me pls?
Online chat or video chat will be great, easy to understand and quickest to do that.
Thanks a lot.

**Comment by swarnava** [comment]
Can you point us your codebase?

**Reply by 6969111** [reply]
Do you mean this?
github.com/6969111/ineedit

**Reply by swarnava** [reply]
Yes, this is a private repo so I am unable to view it. Please make it public :)

**Reply by 6969111** [reply]
I changed to public according your request. Pls proceed.

**Reply by 6969111** [reply]
My project is on React, so in the article you should refer in section 3 to  "Other Frameworks".
![image](https://user-images.githubusercontent.com/112797766/203733648-5a6aee4c-2312-48ea-8f67-dff55f5d1d20.png)

**Reply by 6969111** [reply]
The help I ask for is related to section 2 and section 3 as well in the article.
https://vercel.com/docs/concepts/analytics/audiences/quickstart

**Reply by 6969111** [reply]
In regards to adding the code lines.

**Reply by swarnava** [reply]
Thanks! Could you also share instruction to setup the project?

**Reply by 6969111** [reply]
I don't understand you.

**Reply by 6969111** [reply]
??
